setup.cfg

```diff
@@ -6,6 +6,7 @@ description-file =
 author = PyCQA
 author-email = code-quality@python.org
 home-page = https://bandit.readthedocs.io/en/latest/
+license = Apache-2.0 license
```

Comment:
So I remember later. According to https://packaging.python.org/specifications/core-metadata/#license, this field is to denote licenses that differ from the License Trove classifiers.  It's also used to distinguish a particular version.  Since our classifier just notes as Apache Software License, this field is properly adding the 2.0 context.